Source/Charts/Renderers/LineChartRenderer.swift

```diff
@@ -642,8 +642,8 @@ open class LineChartRenderer: LineRadarRenderer
         for i in 0 ..< dataSets.count
         {
             guard let dataSet = lineData[i] as? LineChartDataSetProtocol else { continue }
-            
-            if !dataSet.isVisible || dataSet.entryCount == 0
+
+            if !dataSet.isVisible || !dataSet.isDrawCirclesEnabled || dataSet.entryCount == 0
```

Comment:
I just saw a performance issue https://github.com/danielgindi/Charts/issues/3774, did you just fix it here?

Reply:
I don't believe the breaking change was ever pulled in.

Reply:
just saw another issue https://github.com/danielgindi/Charts/issues/3798 about this. seems #3798 can be closed if we merged this PR? So I don't have to create another one for master branch